src/test/java/stepsDefinition/MainMemberAt81Years.java

```diff
@@ -0,0 +1,97 @@
+package stepsDefinition;
+
+import PageObjects.*;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import net.thucydides.core.annotations.Steps;
+
+public class MainMemberAt81Years {
+
+
+    @Steps
+    HomePage homePage;
+    @Steps
+    MemberPage memberPage;
+
+    @Steps
+  AdditionalMembers additionalMembers;
+
+
+
+
+    @Given(": Main with Age above restrictions")
+    public void main_with_age_above_restrictions() {
+        homePage.open();
+        homePage.ClickBuyNowFuneralPlans();
+        memberPage.EnterEmail("emailzsdc@gmail.com");
+        homePage.clickContinueBtn();
+    }
+
+    @When(": User captures age above restriction")
+    public void user_captures_age_above_restriction() {
+        memberPage.SelectTitle("Dr");
+        memberPage.EnterName("SOLA");
+        memberPage.Entersurname("SALA");
+        memberPage.enterMemberIDnum("4207297888098");
+        memberPage.enterMobileNum("0825698547");
+        memberPage.enterPostalAddress("21 Jump Street");
+        memberPage.EnterPostalCode("0152");
+        memberPage.selectIncome("R10,000 - R15,000");
+        memberPage.selectOccupation("Technician");
+        memberPage.selectEducation("University, Technikon, Degree, Diploma");
+        memberPage.acceptFICAdecl("Yes");
+        memberPage.acceptFicaDeclaration2("Yes");
+        memberPage.clickContinueBt();
+
+    }
+
+    @Then(": User should get error message.")
+    public void user_should_get_error_message() {
+        memberPage.clickContinueBt();
```

Comment:
add assertion -  this step is a pre condition so you have to check if a error message shows